example/zipkin_like/span.py

```diff
@@ -135,7 +146,7 @@ def is_debug(self):
         return self.trace_context.flags & DEBUG_FLAG == DEBUG_FLAG
 
     def __str__(self):
-        from .encoding import TraceContextEncoder
+        from .marshaling import TraceContextEncoder
```

Comment:
this was just to fix that was left after renaming. We should rename the .py file itself